executor/split_test.go

```diff
@@ -33,20 +34,15 @@ import (
 	"github.com/pingcap/tidb/tablecodec"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/mock"
+	"github.com/stretchr/testify/assert"
 )
 
-var _ = Suite(&testSplitIndex{})
-
-type testSplitIndex struct {
-}
-
-func (s *testSplitIndex) SetUpSuite(c *C) {
-}
-
-func (s *testSplitIndex) TearDownSuite(c *C) {
+func Commentf(format string, args ...interface{}) string {
+	return fmt.Sprintf(format, args)
```

Comment:
I don't think we need this because `assert.Equal` accepts interfaces:

```go
func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool
```